src/Header/Header.tsx

```diff
@@ -1,29 +1,53 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Header.css';
-import logo from '../assets/logo.png';
-import { Navbar, Nav, Button } from 'react-bootstrap';
-import { useHistory } from 'react-router-dom';
+import folderIcon from '../assets/headerIcons/folderIcon.svg';
 
 const Header = () => {
-  const history = useHistory();
+  const [Menu, setMenu] = useState(false);
+
+  function openMenu() {
+    setMenu(!Menu);
+  }
+
   return (
-    <header className="navHeader">
-      <Navbar expand="lg">
-        <Navbar.Brand href="/">
-          <img src={logo} className="logo" alt="Reclass logo" />
-        </Navbar.Brand>
-        <Navbar.Toggle aria-controls="basic-navbar-nav" />
-        <Navbar.Collapse id="basic-navbar-nav">
-          <Nav className="ml-auto">
-            <Nav.Link onClick={() => history.push('/')}>Home</Nav.Link>
-            <Nav.Link href="#about">About</Nav.Link>
-            <Nav.Link href="#contact">Contact Us</Nav.Link>
-            <Button className="btn-reclass" variant="link">Sign In</Button>
-          </Nav>
-        </Navbar.Collapse>
-      </Navbar>
-    </header>
+    <div className='fixed-sidebar' onClick={openMenu}>
+      <div className='fixed-sidebar-options'>
+        <input type='image' alt='' className='fixed-sidebar-icon'src={folderIcon} />
```

Comment:
Please add a space between the `className` and `src`
This would also be where you would add the ReactComponent as the SVG instead of using `src`
Also why is this an input? It looks like it's just acting like a component?

Reply:
I completely changed the navbar so this doesn't apply anymore. Check it out when you have the time!